### PR TITLE
Add new values to the config API endpoint

### PR DESCRIPTION
**Pull request by sharkykh**
Commit messages say it all.

### CODE REVIEW

**Comment by OmgImAlexis** [comment]
Does this allow `/api/v2/config` to return all three sections?

**Comment by OmgImAlexis** [comment]
You'll likely need to update the dredd api description file.

**Comment by sharkykh** [comment]
`GET /api/v2/config` - You get everything (`main`, `qualities`, `statuses`) - the object will be split into the sections.
`GET /api/v2/config/main` - Just one section as the main object.
`GET /api/v2/config/qualities?append=statuses,main` - Get `qualities`, `statuses` and `main` with the same schema as the first call.

**Comment by OmgImAlexis** [comment]
Is that last one really needed? The endpoints should be independent. If you need multiple either get /config or each of them and merge on the client.

**Comment by OmgImAlexis** [comment]
@sharkykh 

`GET /api/v2/config` is returning an array with a single object that contains the configs. Can you please change this to returning just the object there's no need for the outer array.

**Comment by sharkykh** [comment]
This is @ratoaq2 's initial pagination code, I haven't touched that. So not really sure.

**Comment by OmgImAlexis** [comment]
Since I can't ever see a reason to paginate the config I've removed that, how this?

```diff
diff --git a/medusa/server/api/v2/config.py b/medusa/server/api/v2/config.py
index 6aa217beb..e34a2e583 100644
--- a/medusa/server/api/v2/config.py
+++ b/medusa/server/api/v2/config.py
@@ -98,7 +98,7 @@ class ConfigHandler(BaseRequestHandler):
             for section in config_sections:
                 config_data[section] = getattr(DataGenerator, 'data_' + section)()

-            return self._paginate([config_data])
+            return self._ok(data=config_data)

         config_data = getattr(DataGenerator, 'data_' + identifier)()

```

**Comment by sharkykh** [comment]
Yep, that's correct.
I pushed this change and also made sure it still works.

**Comment by OmgImAlexis** [comment]
You may need to change the `store.js` as I think it's still getting the first array item.

File: themes-default/slim/static/js/store.js [hunk 1]
```diff
@@ -233,9 +240,16 @@ const store = new Puex({
     // No actions should write to the store
     // Please use store.commit to fire off a mutation that'll update the store
     actions: {
-        getConfig() {
-            return api.get('/config/main').then(res => {
-                store.commit(ADD_CONFIG, res.data);
+        getConfig(context, section) {
+            return api.get('/config/' + (section || '')).then(res => {
+                if (!section) {
```
Comment (sharkykh) [inline themes-default/slim/static/js/store.js]: That should be `if (section) {` if I'm not mistaken.

**Pull request merged**